### PR TITLE
Viz cli tutorial updated

### PR DESCRIPTION
**Pull request by maharshi-gor**
This PR introduces a CLI tutorial for Viz.

The tutorial talks about
 - Opening a brain image.
 - Opening a brain image with volumes.
 - Opening a tractogram.
 - Opening a surface.

### CODE REVIEW

File: doc/interfaces/viz_flow.rst [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+This tutorial shows how to visualize a tractogram.
+
+Using a terminal, let's download a dataset called ``bundle_atlas_hcp842``. You
+can skip this step if you already have the dataset downloaded. ::
+
+  dipy_fetch bundle_atlas_hcp842
+
+
+Let's try to load the tractogram.
```
Comment (skoudoro) [inline doc/interfaces/viz_flow.rst:103]: Add that we support multiple format (check the list in the codebase)

File: doc/interfaces/viz_flow.rst [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+
+**For macOS and Linux** ::
+
+  dipy_horizon ~/.dipy/bundle_atlas_hcp842/Atlas_80_Bundles/whole_brain/whole_brain_MNI.trk --cluster
```
Comment (skoudoro) [inline doc/interfaces/viz_flow.rst:107]: Why `--cluster`? is it always needed?  Please explain this part

**Comment by maharshi-gor** [comment]
@skoudoro , 
Thank you for your feedback, I have updated the documentation accordingly.
Please do not merge it. I might need to add more content here.

File: doc/interfaces/viz_flow.rst [hunk 3]
```diff
@@ -0,0 +1,163 @@
+.. _viz_flow:
+
+====================================
+Visualization Command Line Interface
```
Comment (skoudoro) [inline doc/interfaces/viz_flow.rst:4]: dMRI Visualization with Horizon

File: doc/interfaces/viz_flow.rst [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+  dipy_horizon ~/.dipy/stanford_hardi/HARDI150.nii.gz
+
+**For Windows(cmd)** ::
+  
+  dipy_horizon %USERPROFILE%/.dipy/stanford_hardi/HARDI150.nii.gz
+
+**For Windows(powershell)** ::
+  
+  dipy_horizon $home/.dipy/stanford_hardi/HARDI150.nii.gz
+
```
Comment (skoudoro) [inline doc/interfaces/viz_flow.rst:89]: Not sure to understand this section, what is the difference with the section `Visualize 3D Brain Image`

File: doc/interfaces/viz_flow.rst [hunk 5]
```diff
@@ -0,0 +1,163 @@
...
+Using a terminal, let's download a dataset called ``bundle_atlas_hcp842``. You
+can skip this step if you already have the dataset downloaded. ::
+
+  dipy_fetch bundle_atlas_hcp842
+
+
+Horizon supports below mentioned tractogram formats.
+
+* .trk
```
Comment (skoudoro) [inline doc/interfaces/viz_flow.rst:105]: - there are much more: `'.trk', '.tck', '.trx', '.vtk', '.vtp', '.fib', '.dpy'`
- Why there is a space between `*`and `.trk` ?

**Review by skoudoro — COMMENTED** [review]
Hi @maharshi-gor,

See my last comment below. Otherwise, this version looks good

File: doc/interfaces/viz_flow.rst (hunk 4 above)
Comment (maharshi-gor) [reply doc/interfaces/viz_flow.rst:89]: If they want to visualize an image with volume

File: doc/interfaces/viz_flow.rst (hunk 5 above)
Comment (maharshi-gor) [reply doc/interfaces/viz_flow.rst:105]: The problem is viz workflow does not support handling .vtk and .vtp and .fib files

**Review by skoudoro — APPROVED** [review]
LGTM, merging

**Pull request merged**